src/core/layout/scrollable-list/scrollable-list.js

```diff
@@ -110,13 +96,24 @@ const setupEvents = (scene, panel) => {
     });
 };
 
-const setupAccessibility = (scene, panel) => {
-    const a11yGroup = scene.add.container();
-    a11yGroup.reset = () => resizePanel(scene, panel);
-    a11yGroup.add(panel);
+export class ScrollableList extends Phaser.GameObjects.Container {
+    constructor(scene) {
+        super(scene, 0, 0);
+        this.panel = createPanel(scene);
 
-    scene.layout.addCustomGroup(scene.scene.key, a11yGroup, 0);
-    a11y.addGroupAt(scene.scene.key, 0);
-};
+        this.add(this.panel);
+        scene.layout.addCustomGroup(scene.scene.key, this, 0);
+        a11y.addGroupAt(scene.scene.key, 0);
+
+        scene.input.topOnly = false;
+        setupEvents(scene, this.panel);
+    }
+
+    reset() {
```

Comment:
You can replace this with 
`
const resizePanel = (scene, panel) => () => {
...

in constructor:
this.reset = resizePanel(this, panel);
```

this should allow you to remove `this.panel` from the class props.

Reply:
I thumbs-up'd this without really understanding what you meant and have since had a think about it- I'll try it and you can tell me if I've made sense of it 😹